packages/editor-ui/tslint.json

```diff
@@ -2,7 +2,8 @@
 	"linterOptions": {
 		"exclude": [
 			"node_modules/**/*",
-			"../../node_modules/**/*"
+			"../../node_modules/**/*",
+			"**/*.json"
```

Comment:
Since the i18n keys are now imported via `resolveJsonModule`, `vue-cli` attempts to lint JSON as JS, breaking the build.